io/polyjson/polyjson.go

```diff
@@ -153,6 +154,82 @@ func Write(sequence Poly, path string) error {
 	return os.WriteFile(path, file, 0644)
 }
 
+// Utilities to convert polyjson objects -> their genbank equivalents
+// TODO add convert <- genbank methods, which is currently difficult as most
+// genbank Meta values are discarded due to lack of support for wildcard metadata in polyjson.
+
+func (sequence *Poly) ToGenbank() genbank.Genbank {
+	gb := genbank.Genbank{
+		Meta:     sequence.Meta.ToGenbank(),
+		Features: make([]genbank.Feature, len(sequence.Features)),
+		Sequence: sequence.Sequence,
+	}
+	for i, f := range sequence.Features {
+		gb.Features[i] = f.ToGenbank()
+		gb.Features[i].ParentSequence = &gb
+	}
+	return gb
+}
```

Comment:
Oh, I see. We have polyjson as a thing. I believe we should nuke it - in `ioToBio` there isn't support for gff, and I don't know of a single instance we actually used the PolyJSON. I think JSON conversion should just follow the closest it can to the actual types. Let's get @TimothyStiles and @carreter opinions on this though.

Reply:
I think is out of scope for this PR, but I will probably end up removing these conversion methods if we plan to deprecate.

Reply:
Makes sense.